pkg/kubectl/cmd/testing/fake.go

```diff
@@ -680,6 +685,7 @@ func testDynamicResources() []*discovery.APIGroupResources {
 			VersionedResources: map[string][]metav1.APIResource{
 				"v1beta1": {
 					{Name: "deployments", Namespaced: true, Kind: "Deployment"},
+					{Name: "thirdpartyresources", Namespaced: false, Kind: "ThirdPartyResource"},
```

Comment:
I did not quite understand this comment 
 > Add another API group stanza for "mygroup.com" with a single version and a single namespaced resource "foos" of kind Foo


In this PR I have also added a test/fixtures/pkg/kubectl/cmd/apply/third-party-resource-entry.yaml file. 
Next I will add unit tests for applying entries to this ThirdPartyResource. 
I think you may be referring to those.

I have responded to your previous comments about this unit test. I hope you do not object the incremental test coverage this adds.